typescript/infra/src/config/funding.ts

```diff
@@ -1,12 +1,20 @@
 import { Contexts } from '../../config/contexts';
+import { KEY_ROLE_ENUM } from '../agents/roles';
```

Comment:
It looks like the file this is imported from did not change in this PR, but I don't feel like an Enum in TS should be SCREAMING_SNAKE_CASE so curious if there is a reason for that choice and if it makes sense to change in this PR.